src/components/List/List.svelte

```diff
@@ -19,7 +19,11 @@
     navigation
   };
 
-  const id = item => item.id || item.value || item.to || item.text || item;
+  const id = item => item.id !== undefined ? item.id :
```

Comment:
Can you please rewrite it as a function in this case? Multiple ternary operators make it hard to understand.

Reply:
No problem! Just pushed this change.